src/parser/grammar.y

```diff
@@ -407,6 +407,13 @@ returnClause(A) ::= RETURN returnElements(B). {
 returnClause(A) ::= RETURN DISTINCT returnElements(B). {
 	A = New_AST_ReturnNode(B, 1);
 }
+// RETURN *
+returnClause(A) ::= RETURN MUL. {
+	A = New_AST_ReturnNode(NULL, 0);
+}
+returnClause(A) ::= RETURN DISTINCT MUL. {
```

Comment:
We've yet to support RETURN DISTINCT, maybe remove this until we do

Reply:
? We support RETURN DISTINCT, though I don't love the implementation - we build a triemap out of `Record_ToString` calls, see the `_encounteredRecord` function in `resultset.c`.

We don't yet support DISTINCT as a modifier in aggregates.

Reply:
Sorry for the confusion